src/SFML/Graphics/ImageLoader.cpp

```diff
@@ -55,7 +56,7 @@ namespace
         sf::InputStream* stream = static_cast<sf::InputStream*>(user);
         return static_cast<int>(stream->read(data, size));
     }
-    void skip(void* user, unsigned int size)
+    void skip(void* user, int size)
```

Comment:
So may I ask again, does stb_image skip back at some point? I mean this function is an internal detail of ImageLoader and won't be used by anyone else.